app/models/warpable.rb

```diff
@@ -1,3 +1,5 @@
+require "mapknitterExporter"
```

Comment:
Hmm, @icarito i'm seeing `File does not exist: mapknitterExporter` from this. I copied the Sinatra app config... any ideas?